nmdc_runtime/api/models/util.py

```diff
@@ -65,8 +176,21 @@ def set_page_if_cursor_unset(cls, values):
 
 
 class PipelineFindRequest(BaseModel):
-    pipeline_spec: str
-    description: str
+    """
+    TODO: Document this class and its attributes.
```

Comment:
Hi @dwinston, if you write a title (alias), description, and example value(s) for each of these two parameters, I'll take care of wrapping them in the necessary FastAPI/Pydantic stuff to get them to appear on the Swagger UI page.

Reply:
see (commit message for) 442d6d420ef3e2bf675bfbcbfe34ba3e8ed693d4